products/ol9/profiles/ism_o.profile

```diff
@@ -71,5 +71,16 @@ selections:
     - '!usbguard_allow_hid_and_hub'
     - '!sshd_allow_only_protocol2'
 
+    - '!package_ypbind_removed'
+    - '!package_ypbind_removed'
+    - '!package_xinetd_removed'
+    - '!ensure_redhat_gpgkey_installed'
+    - '!audit_rules_usergroup_modification_opasswd'
+    - '!audit_rules_usergroup_modification_gshadow'
+    - '!audit_rules_usergroup_modification_passwd'
+    - '!audit_rules_usergroup_modification_group'
+    - '!audit_rules_usergroup_modification_shadow'
+    - '!service_xinetd_disabled'
+
```

Comment:
Could you add these removals:

```
  - '!package_talk_removed'
  - '!package_talk-server_removed'
  - '!package_rsh-server_removed'
  - '!package_rsh_removed'
```

We recently removed them from our e8 profile